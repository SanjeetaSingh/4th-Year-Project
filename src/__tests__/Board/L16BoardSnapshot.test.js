import renderer from 'react-test-renderer'

import Level16 from '../../Level16/L16-Board/L16-Board'

it('snapshot test level 16', () => {
    const tree =  renderer.create(<Level16 />).toJSON()
    expect(tree).toMatchSnapshot();
});