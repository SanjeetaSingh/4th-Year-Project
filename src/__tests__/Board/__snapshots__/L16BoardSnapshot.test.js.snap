// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test level 16 1`] = `
<div
  className="container"
>
  <div
    className="boardcontainer14"
  >
    <div
      class="in"
    >
      <button
        class="buttonIn"
        onClick={[Function]}
        type="submit"
      >
        Instructions
      </button>
      <button
        class="buttonBone"
        onClick={[Function]}
        type="submit"
      >
        Bone Class
      </button>
    </div>
    <div
      className="board16"
    >
      <div
        className="box"
        data-col={0}
        data-row={0}
      />
      <div
        className="box"
        data-col={1}
        data-row={0}
      />
      <div
        className="box"
        data-col={2}
        data-row={0}
      />
      <div
        className="box"
        data-col={3}
        data-row={0}
      />
      <div
        className="box"
        data-col={4}
        data-row={0}
      />
      <div
        className="box"
        data-col={0}
        data-row={1}
      />
      <div
        className="box"
        data-col={1}
        data-row={1}
      />
      <div
        className="box"
        data-col={2}
        data-row={1}
      />
      <div
        className="box"
        data-col={3}
        data-row={1}
      />
      <div
        className="box"
        data-col={4}
        data-row={1}
      />
      <div
        className="box"
        data-col={0}
        data-row={2}
      />
      <div
        className="box"
        data-col={1}
        data-row={2}
      >
        <img
          alt="d"
          class="goal"
          id="food"
          src="assets/bone.png"
        />
      </div>
      <div
        className="box"
        data-col={2}
        data-row={2}
      />
      <div
        className="box"
        data-col={3}
        data-row={2}
      />
      <div
        className="box"
        data-col={4}
        data-row={2}
      />
      <div
        className="box"
        data-col={0}
        data-row={3}
      />
      <div
        className="box"
        data-col={1}
        data-row={3}
      />
      <div
        className="box"
        data-col={2}
        data-row={3}
      >
        <img
          alt="d"
          class="paw"
          id="paw"
          src="assets/paw.png"
        />
      </div>
      <div
        className="box"
        data-col={3}
        data-row={3}
      />
      <div
        className="box"
        data-col={4}
        data-row={3}
      />
      <div
        className="box"
        data-col={0}
        data-row={4}
      />
      <div
        className="box"
        data-col={1}
        data-row={4}
      />
      <div
        className="box"
        data-col={2}
        data-row={4}
      />
      <div
        className="box"
        data-col={3}
        data-row={4}
      >
        <img
          alt="d"
          class="image"
          id="dog"
          src="assets/dog.png"
        />
      </div>
      <div
        className="box"
        data-col={4}
        data-row={4}
      />
    </div>
  </div>
  <div
    className="bottom-panel"
  >
    <div
      class="border"
    >
      <div
        class="level16Contain"
      >
        <h2>
          Level 16:
        </h2>
        <div
          class="borderPanel"
        >
          <div
            class="toppart16"
          >
            <div
              class="containing"
            >
              <p
                class="number"
                data-testid="counter"
                id="count"
              >
                0/5
              </p>
            </div>
            <div />
            <button
              class="seemingly-inner-button-hint16"
              disabled={false}
              onClick={[Function]}
            >
              Hint
            </button>
            <p
              class="titles"
            >
              Enter Sequence:
            </p>
            <div
              class="buttons-wrapper12"
            >
              <button
                class="seemingly-inner-button12"
                disabled={false}
                onClick={[Function]}
              >
                <i
                  class="fa fa-info"
                />
                    Variables
              </button>
              <p
                class="subtitles"
              >
                Variable Declaration
              </p>
            </div>
            <div>
              <p
                class="bones"
              >
                Bone bone = new Bone()
              </p>
            </div>
            <form
              class="var"
            >
              int row =  
              <input
                class="input"
                id="var1"
                onChange={[Function]}
                size="10"
              />
                 ; 
              <br />
              <br />
              int col = 
              <input
                class="input"
                id="var2"
                onChange={[Function]}
                size="10"
              />
                ;  
              <br />
              <br />
                int steps =  
              <input
                class="input"
                id="var3"
                onChange={[Function]}
                size="10"
              />
                ;
              <br />
              <br />
            </form>
            <p
              class="lists"
              id="action"
            />
          </div>
        </div>
        <div
          class="outside16"
        >
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Left
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Up
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Down
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Right
          </button>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              For
            </button>
          </div>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              If Statement
            </button>
          </div>
          <button
            class="button"
            onClick={[Function]}
            type="submit"
          >
            bone.found()
          </button>
          <button
            class="buttonClear"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Clear
          </button>
          <button
            class="buttonSub"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
             Submit
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
